src/openms_gui/source/VISUAL/SpectraSelectionTabs.cpp

```diff
@@ -103,9 +103,9 @@ namespace OpenMS
     });
 
     SpectrumCanvas* cc = tv_->getActiveCanvas();
-    int layer_row = (cc == nullptr ? -1 : (int)cc->activeLayerIndex());
+    Size layer_row = (cc == nullptr ? -1 : cc->getCurrentLayerIndex());
 
-    if (cc == nullptr || layer_row == -1)
+    if (cc == nullptr || layer_row == (Size)-1)
```

Comment:
is this test redundant?
e.g., test only for layer_row == (Size)-1 should suffice

Reply:
good point. looking a bit closer, layer_row is not required at all..

Reply:
looking even closer, it is...